apply-labels/readme.md

```diff
@@ -0,0 +1,98 @@
+# Apply labels as labeler from file.
+
+This folder focuses on workflows for row-labeling, where labelers' have applied labels externally and you need to want to import them to Datasaur and view the [analytics](https://docs.datasaur.ai/workspace-management/analytics).
```

Comment:
`need to want to` keknya bisa pilih salah satu aja?

Reply:
kayanya kemarin waktu nulis ada `/` nya deh
hilang ke mana ya

anyway, agree jadi satu aja lebih concise